src/main/java/com/minecolonies/entity/ai/util/Structure.java

```diff
@@ -27,6 +27,9 @@
      */
     public static final class StructureException extends Exception
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this class from 26 to the maximum allowed 25 or externalize it in a public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2972)